spack_dlaf/packages/lapackpp/package.py

```diff
@@ -1,13 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
 from spack import *
 
+
 class Lapackpp(CMakePackage):
     """LAPACK++: C++ API for the Basic Linear Algebra Subroutines (University of Tennessee)"""
 
     homepage = "https://bitbucket.org/icl/lapackpp"
     hg       = "https://bitbucket.org/icl/lapackpp"
-    maintainers = ['Sely85']
+    maintainers = ['teonnik', 'Sely85']
 
-    version('develop', hg=hg)
+    version('develop', hg=hg, revision="7ffa486")
 
     depends_on('blaspp')
```

Comment:
In principle yes. If blaspp is built with a library that doesn't provide lapack, then lapackpp might break (I am not sure). However, currently the recipe lacks a way to build lapackpp with a custom LAPACK provider (other than the one from blaspp) so the option would be moot. Once the recipe is updated, the depends clause can be added, for now, lets just leave it as it is to avoid breaking things.